wire-ios-sync-engine/Source/Use cases/EnableAnalyticsUseCase.swift

```diff
@@ -33,15 +33,18 @@ public protocol EnableAnalyticsUseCaseProtocol {
 
 /// Concrete implementation of the EnableAnalyticsUseCaseProtocol.
 /// This struct is responsible for enabling analytics sharing for a specific user profile.
```

Comment:
Same comments in this file as I left in `EnableAnalyticsUseCase`

Reply:
Removed the generics from the use cases: https://github.com/wireapp/wire-ios/commit/c82f5d3b1e3b7847902aa182978145fcd4e241d5